Library/Homebrew/cask/cmd/upgrade.rb

```diff
@@ -93,6 +93,17 @@ def self.upgrade_casks(
           end
         end
 
+        manual_installer_casks = outdated_casks.select do |cask|
+          cask.artifacts.any?(Artifact::Installer::ManualInstaller)
+        end
+
+        if manual_installer_casks.present?
+          count = manual_installer_casks.count
+          ofail "Not upgrading #{count} `installer manual` #{"cask".pluralize(count)}."
```

Comment:
made a variable `count` to avoid `brew style`warning `Line is too long.`.
```bash
 C: Line is too long. [133/118]
          ofail "Not upgrading #{manual_installer_casks.count} `installer manual` #{"cask".pluralize(manual_installer_casks.count)}."
                                                                                                                      ^^^^^^^^^^^^^^^
```